Tests/Acceleromter_Tests.cs

```diff
@@ -21,5 +21,30 @@ public Accelerometer_Tests()
         [Fact]
         public void IsMonitoring_Default_On_NetStandard() =>
             Assert.False(Accelerometer.IsMonitoring);
+
+        [Fact]
+        public void AccelerometerData_Equals_AreSameCopy()
+        {
+            var data = new AccelerometerData(0, 0, 0);
+            var copy = data;
+            var res = data.Equals(copy);
+            Assert.True(res);
```

Comment:
should also add == and !=

Reply:
I think if you just add the == and != then it should be closer to good to go.

Reply:
I believe you looked at an old version of this, at the time of writing the class already has unit tests for op_equality and op_inequality. line 51 and following